internal/metamorph/processor.go

```diff
@@ -317,6 +317,7 @@ func (p *Processor) updateMined(ctx context.Context, txsBlocks []*blocktx_api.Tr
 	}
 }
 
+// start processing txs submitted to message queue
```

Comment:
Change comment to 

`// StartProcessSubmittedTxs starts processing txs submitted to message queue`